core/arch/arm/kernel/kern.ld.S

```diff
@@ -82,11 +85,11 @@ SECTIONS
 #endif
 		*(.sram.text.glue_7* .gnu.linkonce.t.*)
 		. = ALIGN(8);
-		__text_end = .;
 	}
+	__text_end = .;
 
-	.rodata : ALIGN(8) {
-		__rodata_start = .;
+	__rodata_start = .;
```

Comment:
I have no strong opinion on this, and it may be a matter of personal taste, but I'd rather keep an align statement here for clarity (although it's not strictly needed since there is one in the previous block). In fact, I like the previous syntax better (it's clear and compact).
Same below of course.